src/web/routes/application/steps/decision/get-decision-status.test.js

```diff
@@ -86,7 +85,7 @@ const DUPLICATE_RESULT = undefined // A DUPLICATE response from the claimant ser
 
 test('getDecisionStatus() should return undefined if verification result has no matching decision', (t) => {
   // TODO these non matching results will need to be updated as stories for epic 'receive my decision' are completed
-  const nonMatchingResults = [{}, SUCCESSFUL_RESULT, PENDING_RESULT_PHONE_OR_EMAIL_MISMATCH]
+  const nonMatchingResults = [SUCCESSFUL_RESULT]
   nonMatchingResults.forEach(verificationResult => {
```

Comment:
Minor - we can remove the `forEach` now.

Reply:
Done.